src/App.tsx

```diff
@@ -1,40 +1,14 @@
-import { useState } from 'react';
+import CreateEvent from './components/CreateEvent/CreateEvent';
 import PageEvent from './components/ PageEvent/PageEvent';
-import moment from 'moment-timezone';
+import { parseUrl } from './utils';
+import styles from './App.module.scss';
 
 function App() {
-  const [eventName, setEventName] = useState('');
-  const [eventDate, setEventDate] = useState('');
-  const [eventTime, setEventTime] = useState('');
-  const [generatedUrl, setGeneratedUrl] = useState('');
-
-  const handleGenerateUrl = () => {
-    const timestamp = moment.tz(`${eventDate} ${eventTime}`, moment.tz.guess()).valueOf();
-    const timezone = moment.tz.guess();
-    const url = `http://url/?ts=${timestamp}&tz=${timezone}&event=${encodeURIComponent(eventName)}`;
-    setGeneratedUrl(url);
-  };
+  const { date, timezone } = parseUrl(window.location.href);
 
   return (
-    <div>
-      <h1>Создание события</h1>
-      <input
-        type="text"
-        value={eventName}
-        onChange={(e) => setEventName(e.target.value)}
-        placeholder="Название события"
-      />
-      <input type="date" value={eventDate} onChange={(e) => setEventDate(e.target.value)} />
-      <input type="time" value={eventTime} onChange={(e) => setEventTime(e.target.value)} />
-      <button onClick={handleGenerateUrl}>Сформировать ссылку</button>
-      {generatedUrl && (
-        <div>
-          <p>
-            Ссылка на событие: <a href={generatedUrl}>{generatedUrl}</a>
-          </p>
-        </div>
-      )}
-      <PageEvent eventUrl={generatedUrl} />
+    <div className={styles.wrapper}>
+      {date && timezone ? <PageEvent eventUrl={window.location.href} /> : <CreateEvent />}
```

Comment:
```suggestion
      {date && timezone ? <PageEvent data={data} timezone={timezone} /> : <CreateEvent />}
```

Instead of URL you might pass date and timezone since they are already parsed. No need to parse the URL again inside that component